src/apps/js_generic/js_generic_base.cpp

```diff
@@ -98,6 +98,23 @@ namespace ccfapp
         policy_name = get_policy_name_from_ident(user_cose_ident);
         id = user_cose_ident->user_id;
         is_member = false;
+
+        auto cose = ctx.new_obj();
+        cose.set(
+          "content",
+          ctx.new_array_buffer_copy(
+            user_cose_ident->content.data(), user_cose_ident->content.size()));
+        cose.set(
```

Comment:
If we are able to avoid copies somehow that would be great. I am not sure we want re-copy the envelope in particular, since that contains the whole content right now (and for the foreseeable future).

Reply:
> the JS runtime now owns the buffer, which is not what we want I don't think?

No, definitely. What would be ideal is a non-owning read-only ArrayBuffer, and if there isn't one I think it might be worth making one in quickjs, for many reasons (eg. large KV values).

> We could also only pass content but not envelope to minimise the number of bytes copied?

Let's do that for now, it's the user scenario we have. We can revisit this later and add other things if needed.